README.md

```diff
@@ -35,62 +35,56 @@ For building documentation:
 ## Build and Install the SLEIGH Library
 
 ```sh
-# Clone this repository (CMake project for SLEIGH)
+# Clone this repository (CMake project for sleigh)
 git clone https://github.com/lifting-bits/sleigh.git
 cd sleigh
 
 # Configure CMake
 cmake -B build -S .
 
 # Build SLEIGH
-cmake --build build -j
```

Comment:
I thought this was using the number of logical cores automatically. Do we need to specify 8 like that?

Reply:
> using the number of logical cores automatically

From the manpage [here](https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-a-project), under `--parallel [<jobs>], -j [<jobs>]`:

> If `<jobs>` is omitted the native build tool's default number is used.

And GNU Make has [no limit on the number of jobs](https://www.gnu.org/software/make/manual/html_node/Parallel.html) to run in parallel if `<jobs>` is not specified:

> If there is nothing looking like an integer after the ‘-j’ option, there is no limit on the number of job slots.

Reply:
Thanks for clarifying. That makes sense.